### PR TITLE
fix(react): update lastMessageUpdate consistently with Webchat Settings

### PR DESCRIPTION
**Pull request by vanbasten17**
## Description
* Update logic to be consistent when resending WebchatSettings.

Linked PR: https://github.com/metis-ai/hubtype-backend/pull/552

### CODE REVIEW

**Review by asastre — COMMENTED** [review]
Since `WebchatSettings` is not a message by itself I don't know if it make sense to update the last message date. What do you think?

**Comment by vanbasten17** [comment]
@asastre it's true that is not a message itself, but when resending unread messages from backend, the last message update date should be taken into account in order to not send additional WebchatSettings messages.

**Review by asastre — APPROVED** [review]

File: packages/botonic-react/src/webchat/webchat.jsx [hunk 1]
```diff
@@ -581,6 +581,7 @@ export const Webchat = forwardRef((props, ref) => {
     updateWebchatSettings: settings => {
       const themeUpdates = normalizeWebchatSettings(settings)
       updateTheme(merge(webchatState.theme, themeUpdates), themeUpdates)
+      updateLastMessageDate(new Date())
```
Comment (dpinol) [inline packages/botonic-react/src/webchat/webchat.jsx:584]: does this depend on #1360? there you're sending new Date().toISOString

**Review by dpinol — CHANGES_REQUESTED** [review]

File: packages/botonic-react/src/webchat/webchat.jsx (hunk 1 above)
Comment (vanbasten17) [reply packages/botonic-react/src/webchat/webchat.jsx:584]: yes, it is related and it depends on it (but this one was created before). Changing this PR to draft

**Comment by vanbasten17** [comment]
@asastre , you were right, there is no need to update the lastMessageDate. When doing the auth call, last message id will be the previous one just before the WebchatSettings in DB. So I'm closing this PR atm. It's a backend issue.

**Pull request closed without merging**